addons/main/functions/fnc_findBuildings.sqf

```diff
@@ -37,10 +37,9 @@ private _housePos = [];
     _housePos append (_house buildingPos -1);
     if (_findDoors) then {
         {
-            if ("door" in toLower (_x)) then {
-                _housePos pushBack (_house modelToWorld (_house selectionPosition _x));
-            };
-        } forEach (selectionNames _house);
+            if !("door" in _x) then {continue};
```

Comment:
why do we use a continue here and additionally a negation. wouldn't it be just simpler to move the pushback into the if? like it was before.

Reply:
Because in my opinion less indentation is easier to read. Is there a specific reason you want it exactly like it was before?

Reply:
in the end i expect negating and an extra script command maybe slower than that plus I personally dislike how it currently reads to be correctly formated it would occupy 3 lines and that just reads worse for me in total plus is unneeded longer
```sqf
if !("door" in _x) then {
    continue;
};
```

Reply:
Fair point with the speed, benchmarked the ``continue`` method to be ~0.05ms (rougly 5%) slower than the original.

But what do you mean correctly formatted? One line if statements are all over this project, some of them over a hundred characters long.